### DISCUSSION TITLE
spacy benchmarks CPU/GPU type

### DISCUSSION THREAD

**Discussion by saareliad** [explosion/spaCy#9624] [Help: Other Questions] [answered]
Hi,
on which CPU did you run 
https://spacy.io/usage/facts-figures#benchmarks-speed
?

similarly, on which GPU did you benchmark?

I wonder what performance should I expect on a single core (single thread) for the `en_core_web_lg`.

I found it quite hard to interpret the results e.g., see how much time was spent at every stage. Do you provide a tool for this?

**Comment by polm** [comment, marked as answer]
We'll need to check what hardware that was run on, but since the [source for the benchmark is available](https://github.com/explosion/projects/tree/v3/benchmarks/speed) it should be easy to run on your own machine and see how it does. The purpose of the benchmarks is not to tell you what throughput to expect on certain hardware, but rather to compare speed to similar libraries.

The breakdown is not by component and we don't have tools for that, sorry. Again, since the benchmark is primarily intended as a comparison with similar libraries, which usually don't have the same component breakdown, end-to-end comparison is more meaningful.

**Comment by svlandeg** [comment]
Hi! As Paul mentioned, the takeaway from these numbers is really the comparison.

That said, the speed benchmark as detailed in the docs was run on an AWS g4dn.2xlarge machine - cf https://aws.amazon.com/ec2/instance-types/g4/ for the instance details.
